pelicanconf.py

```diff
@@ -1,3 +1,29 @@
-# -*- coding: utf-8 -*-
+#!/usr/bin/env python
+# -*- coding: utf-8 -*- #
```

Comment:
this line should be just `# -*- coding: utf-8 -*-` - that's the first time I'm seeing tailing `#`.